src/main/java/by/onliner/cucumber/steps/CatalogSteps.java

```diff
@@ -0,0 +1,23 @@
+package by.onliner.cucumber.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static by.onliner.core.utils.EqualsUtil.equalContains;
+
+public class CatalogSteps extends AbstractSteps{
+
+    @When("Input {string} in search input field")
+    public void inputTextInSearchField(String text) {
+        mainPage.getMenu().inputTextInSearchField(text);
+    }
+
+
+    @Then("Searched item name will contain {string}")
```

Comment:
Update step description as well, it should represent an action and all the steps should be specified in the same style
ex. 'Verify searched item contains {string}'

Reply:
Fixed